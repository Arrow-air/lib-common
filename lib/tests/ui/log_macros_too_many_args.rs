//! Test too many arguments for log_macros

use lib_common::log_macros;

log_macros!("one", "two", "three");

fn main() {}
