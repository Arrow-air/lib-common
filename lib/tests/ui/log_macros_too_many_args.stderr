error: log_macros expects either one or two arguments
 --> tests/ui/log_macros_too_many_args.rs:5:13
  |
5 | log_macros!("one", "two", "three");
  |             ^^^^^^^^^^^^^^^^^^^^^
